README.md

```diff
@@ -10,6 +10,9 @@ powered by Exonum.
 The project is at alpha stage. Currently you can define a stateful Exonum Service 
 in Java, configure a network of nodes and run the service in it.
 
+If you are using Exonum in your project and want to be listed on our website & GitHub
```

Comment:
I'd move it below the list — it is the least relevant information at the moment.